YTSpammerPurge.py

```diff
@@ -673,16 +676,16 @@ def primaryInstance(miscData):
           if len(videosToScan) < numVideos:
             print(f"\n{F.YELLOW} WARNING:{S.R} Only {len(videosToScan)} videos found. Videos may be skipped if there are no comments.")
           print("\nRecent Videos To Be Scanned:")
-          for i in range(len(videosToScan)):
+          for i, video in enumerate(videosToScan):
             if config['skip_confirm_video'] == False:
               if i == 10 and len(videosToScan) > 11:
                 remainingCount = str(len(videosToScan) - 10)
                 userChoice = choice(f"There are {remainingCount} more recent videos, do you want to see the rest?")
                 if userChoice == False:
                   break
                 elif userChoice == None:
-                  return True # Return to main menu
-            print(f"  {i+1}. {videosToScan[i]['videoTitle']}")
+                  return True # Return to main menu         
```

Comment:
Absolutely a tiny thing (and not important at all,) my OCD compels me to point out the extra spaces added here.

Reply:
ok???

Reply:
> Absolutely a tiny thing (and not important at all,) my OCD compels me to point out the extra spaces added here.

I mean, he removed it from the code, so I guess it's fine.

Reply:
Not these ones @TechStudent11 
![image](https://user-images.githubusercontent.com/34950371/177227356-bc8c3bde-2592-47a6-a65f-9a49194b506a.png)

Reply:
> Not these ones @TechStudent11 ![image](https://user-images.githubusercontent.com/34950371/177227356-bc8c3bde-2592-47a6-a65f-9a49194b506a.png)

OH GOSH WHY

Reply:
???

Reply:
updates?

Reply:
updates?

Reply:
updates?

Reply:
He is clearly busy.

Reply:
updates???????????

Reply:
> He is clearly busy.

@DinhHuy2010

Reply:
@DinhHuy2010 the more you ask for updates, the longer it will take to be reviewed by @ThioJoe :) , don't be impatient, some PR's I've seen are open for 2-5 years, so ThioJoe will surely review when he got time man

Reply:
ok

Reply:
it since 1 week and 5 days

Reply:
> it since 1 week and 5 days

bro, just stop. 
or
@ThioJoe will just not review this PR and close it. He will have a look when he's free. It's his project.

Reply:
Plus, as per your username, you are born in **2010**, which means you are 12 years OLD. 

You require to be at least 13+ to be using GitHub or any social media.

Reply:
I don't think having a year in your username means that year is your year of birth. Could just be a very important year to them.